test/cpp/jit/test_gpu.cpp

```diff
@@ -18822,6 +18822,318 @@ TEST(NVFuserTest, FusionRfactorPredication1_CUDA) {
       &fusion, cg_outputs, {at_t0, at_t3}, {at_t2, at_t4}, __LINE__, __FILE__);
 }
 
+TEST(NVFuserTest, FusionRfactorPredication2_CUDA) {
```

Comment:
This was a test created for #1250 but removed as it was failing due to the extent problem.